### ISSUE TITLE
WE NEED A NAME

### ISSUE THREAD

**Issue by andrewMacmurray** [foundersandcoders/freelance-website#9]
pleeeaaasssee put ideas here

**Comment by sohilpandya** [comment]
FAC-Tree 👊

**Comment by eliasmalik** [comment]
For inspiration (or laziness): https://www.namemesh.com/company-name-generator

**Comment by tormod17** [comment]
FAC DEM

**Comment by rub1e** [comment]
How are you ordering your priorities? Cos I'd say this is the very lowest priority imaginable. 

But, OK:
- reFACtor
- FACbook
- FACUK
- Strapline: The FAC that shipped a thousand launches

**Comment by andrewMacmurray** [comment]
@rub1e it's a website not an app so branding is more of a bottleneck (can't make illustrations / other assets without a brand direction)

Also no puns 😄 (Dan's orders)

**Comment by rub1e** [comment]
Indeed - but your name should flow from your brand identity and not the other way around

**Comment by sofer** [comment]
The Founders & Coders Agency
The Freelance Factory
Factory Apps
Coders for Founders

**Comment by iteles** [comment]
@sofer I thought the whole point was for the freelancers to be _separate_ to FAC. Why would we call it 'The Founders & Coders Agency'? Or FAC-anything?
I don't feel strongly about the FAC-something names, I'm just raising the question.

**Comment by andrewMacmurray** [comment]
@iteles good point, would be nice to find some way of linking them (maybe more implicitly) don't want to forget our roots 😆

**Comment by sohilpandya** [comment]
Pollen.io
New milk.io
Bolt and gay.co.uk

**Comment by andrewMacmurray** [comment]
@sohilpandya I love pollen.io! It's a shame it's taken but I'm sure we could think of something around it

**Comment by sofer** [comment]
@iteles perhaps separate enough that the freelance agency has its own identity, but close enough that the connexion is clear.

**Comment by sohilpandya** [comment]
Stacked.io

**Comment by tormod17** [comment]
FAC STACK

**Comment by oturnermajor** [comment]
@andrewMacmurray pollenate.io? Doesn't seem to be taken.

**Comment by eliasmalik** [comment]
cradle
- Because we're babies (relatively speaking)
- Because we nurture new ideas (i.e. build MVPs for startups)

**Comment by sohilpandya** [comment]
I like both pollinate and cradle 

Along with FACTREE

**Comment by mcelearr** [comment]
FACtree is a strong one for me.
Also, how about these:
- Founders and Coders Labs (FAC Labs)
- FAC Foundry

**Comment by rub1e** [comment]
https://github.com/foundersandcoders/freelance-website/issues/9#issuecomment-221222213

**Comment by sohilpandya** [comment]
Hey @/all, Just wanted to come to conclusion with regards to the name, we've put majority of the names that have been listed above to the group and FACTREE has got the most positive reactions, people seem to like it! And unlike some of the other names that would not have a meaning behind it, we can explain to potential clients what FACTREE actually means.

Hence we were rolling with that name. 

I understand the hesitation with using FAC in the name, but a lot of us feel that as we are still in the space and majority of the work that we receive still comes from FAC (either hello@f&c.com or phone), FACTREE was a good option. 

Does anyone feel strongly about not using the above name and have an alternative that works? We've been at this for a while and its about time we moved on to actually working on the other aspects of the site. @heron2014 has made a great start on designing some logos [here](https://github.com/fac-freelancers/website/issues/20#issuecomment-228607698)

**Comment by rub1e** [comment]
Sorry mate, but FACTREE is a terrible name

It's a pun based on the fact that we call Founders and Coders _FAC_, but no one knows that. Most external people actually call it F&C

It's a pun based on the word _Factory_ which is not a good association (because factories involve impersonal automata which churn out identical products on an industrial scale)

It's also a pun based on the FAC's own logo, which has a tree in it. 

So when you say 

> And unlike some of the other names that would not have a meaning behind it, we can explain to potential clients what FACTREE actually means.

... what _does_ it actually _mean_? It's like you've watched that episode of the Simpsons and said "We need a name that's witty at first, but that seems less funny every time you hear it". But [The Bee Sharps](https://www.youtube.com/watch?v=7NEWA32jqk0) was actually quite funny. 

The point of raising the question of using the FAC name wasn't about whether you feel it's relevant to you - it's more about the risk to the main FAC business of your using the name. What if you reflect poorly on FAC? All of your successes already belong to FAC because it taught you coding in 16 weeks for free - why expose FAC to all of your potential failures as well? At the very least, actually have the debate!!

And as for moving on to other aspects of the site - JUST DO IT. How could you allow a lack of a decent  name to stop you from working on something which will provide you with a livelihood? JUST GET ON WITH IT.

The only positive thing about the name FACTREE is that it might allow you to finish a single-page static website that was started in the first week of May and shows no signs of being done before the clock hits two months. **But already, you're failing to live up to the implications of calling yourselves a factory.**

Go with the meaningless FACTREE if you like - just make sure you're choosing it for a reason rather than because acquiescing is easier and Anita's already done some lovely logos for it and it's one fewer decision for you to confront. Either way, don't let it stop you from finishing the work.

**Comment by sohilpandya** [comment]
You've made a valid point with regards to being slow in getting the site up and running, I'll take the blame for that. I've been pretty busy with a project and then QMUL, but I have got a week off now and will be coming up with something pretty soon.

When you say the following: 

> The point of raising the question of using the FAC name wasn't about whether you feel it's relevant to you - it's more about the risk to the main FAC business of your using the name. What if you reflect poorly on FAC?

When you are talking about the business model, what exactly are you referring to? 
I thought that the freelancers are a big part of the business model, as our contributions from the ongoing projects is a reflection of the success of what can be achieved after completing a 16 week course.

Getting back to the name, for now we will be sticking with FACTREE, we'll always have some people who will dislike it, but let's just get on with it, as the majority of the people that I've spoken to are in favour of the name.

**Comment by rub1e** [comment]
@sohilpandya there is no yogic contortion of the imagination that makes you solely to blame for this mate!

Re name - like I say, if it gets you over the finish line before we invoke article 50, I'm happy. Well, you know...

Re biz model - FAC doesn't actually have one yet. I was more worried about reputation. By inextricably linking yourself to the FAC name, you affect its reputation. Self-determination and self-sufficiency should mean that if you screw up, your reputation is tarnished, but FAC is somewhat insulated. I did think it was important for everyone to understand the implications of this and debate it before making that decision.

**Comment by otajor** [comment]
So up until now, pretty much all FAC freelance projects (not counting DWYL) have been very closely linked to the FAC name, in that they were carried out by FAC as a coding cooperative, under the FAC name.

@rub1e can you maybe give some examples of the kind of reputational damage that has come about in the past because of this association between the freelance projects and FAC as a coding bootcamp? It would be good to know the details about the extent and nature of that damage and its short or long-term impacts on FAC.

Just trying to get an informed view of what you see as the potential risk here.

**Comment by rub1e** [comment]
Well, my hope was to have this discussion before the decision was made, but hey-ho.

Even though we have had some dissatisfied customers in the past - who've ended up not paying and bad-mouthing us etc. - there is no way to quantify the reputational damage. Such damage happens at meetups, when one entrepreneur tells another about her bad experience over cardboard pizza and free beer. But that's also exactly where WOM recommendations happen. People talk...

On a more conceptual level, the freelance agency is supposed to be a separate business with a separate website and separate governance, yes? So calling it FAC-anything sends mixed messages from the start. 

I think it's important to realise that all businesses screw up from time to time - even ones whose workers have more than four months' experience in their field. When you do screw up - and you will - I don't think it's fair that it should come back to the FAC Academy.

**Comment by heron2014** [comment]
I just came up with these names:
- **techfueled.com** or co.uk or io - all available
- **tecfueled.com** or co.uk
- **tecfuel**.co.uk 

We could reuse the TF logo :)
